packages/web-components/fast-components/src/radio-group/radio-group.ts

```diff
@@ -8,6 +8,11 @@ import {
 } from "@microsoft/fast-web-utilities";
 import { RadioControl } from "../radio";
 
+export enum RadioOrientation {
```

Comment:
Don't we have some kind of orientation enum in web-utilities?

Reply:
It would be nice to have a single Orientation used everywhere. If we don't have it, let's create that and use it here and in the other web components. Good call @janechu !

Reply:
Yea I did mention this on another PR or in discord but I'm not sure if we do or not, I will use the one in web-utilities or make it if it doesn't exist

Reply:
yea I don't see one or any existing module that makes sense to put it in.

Reply:
oh we do, it's in aria.ts, ooook

Reply:
done